Source/DafnyLanguageServer.Test/ProjectFiles/ProjectFilesTest.cs

```diff
@@ -77,13 +79,18 @@ public class ProjectFilesTest : ClientBasedLanguageServerTest {
   }
 }
 
+function Zoo(): set<(int,int)> {
+  set x: int | 0 <= x < 5, y | 0 <= y < 6 :: (x,y)
+}
+
 ghost function Bar(): int { 3 }".TrimStart();
 
     var projectFileSource = @"
 includes = [""**/*.dfy""]
 
 [options]
 warn-shadowing = false
+quantifier-syntax = 4
```

Comment:
In your test, could you please add an option that was otherwise translated to an enum or otherwise processed?
For example:
test-assumptions = externs
It serves no purpose for your example other than testing that the parsing mechanism works (which it did not until this PR)

Reply:
`quantifier-syntax` is an enum type option

Reply:
Oh great, you got my point and that's perfect then.